word_frequency.py

```diff
@@ -0,0 +1,60 @@
+import re
+import sys
+
+
+def clean_up(word):
+    '''Removes puncuation from each word.'''
+    word = re.sub(r'[^A-Za-z]', "", word).lower()
+    # characters = (".", ",", ":","?","!","&", "*", "\"")
+    # dash = ("-")
+    # for char in characters:
+    # word = word.replace(char, "").lower()
+    return word
+
+
+def dict_adder(words):
+    """Adds and counts all the words from the string into the dictionary."""
+    text = words.split()
+    for word in text:
+        word = clean_up(word)
+        if word == '':
+            pass
+        elif word in word_dict:
+            word_dict[word] += 1
+        else:
+            word_dict[word] = 1
+    return word_dict
+
+
+def word_frequency(long_string, counter=1, num_its=20):
```

Comment:
The problem you're having with making the tests pass is that this function should return a dictionary of words -> number of times they were used. The printing logic should be in a separate function that takes the return value of this function. [Re-read the `README` file](https://github.com/tiyd-python-2015-01/word-frequency#requirements) for instructions on this.